src/main/java/codeu/controller/ActivityFeedServlet.java

```diff
@@ -64,7 +69,34 @@ void setMessageStore(MessageStore messageStore) {
     this.messageStore = messageStore;
   }
 
-  /**
+  public static class DateFormatter {
+
+    static String format = "EEE MMM dd yyyy hh:mm:ss a z";
```

Comment:
Use final for constants, and capitalize the name
i.e. static final String FORMAT

Reply:
Done